app/ide-desktop/lib/content/src/remoteLog.ts

```diff
@@ -49,24 +51,26 @@ export async function remoteLog(
     message: string,
     metadata: unknown
 ): Promise<void> {
-    try {
-        const headers: HeadersInit = [
-            ['Content-Type', 'application/json'],
-            ['Authorization', `Bearer ${accessToken}`],
-        ]
-        const body = JSON.stringify({ message, metadata })
-        const response = await fetch(REMOTE_LOG_URL, { method: 'POST', headers, body })
-        if (!response.ok) {
-            const errorMessage = `Error while sending log to a remote: Status ${response.status}.`
-            try {
-                const text = await response.text()
-                throw new Error(`${errorMessage} Response: ${text}.`)
-            } catch (error) {
-                throw new Error(`${errorMessage} Failed to read response: ${String(error)}.`)
+    if (REMOTE_LOG_URL != null) {
```

Comment:
Personally I'm a fan of early return rather than [if-pyramids](https://en.wikipedia.org/wiki/Pyramid_of_doom_(programming)) but I suppose that's a stylistic choice.

Reply:
it's enforced by lints, but i agree. i kinda assumed we wanted to avoid early return because (i think) that's wojciech's preference, but we can definitely remove the lint enforcing if-pyramids - it's not like it's particularly useful. perhaps we can enforce that early return is only used at the very start though - because i guesss the point was that having returns in the *middle* of the function makes behavior surprising since you can easily miss it in a long function

Reply:
I think I do recall Wojciech prefers them, so I suppose it can stay 🤷

Reply:
@wdanilo care to weigh in?